Directory.Packages.props

```diff
@@ -0,0 +1,93 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <!-- TFM agnostic -->
+  <ItemGroup>
+    <PackageVersion Include="Bogus" Version="34.0.2" />
+    <PackageVersion Include="CorrelationId" Version="3.0.1" />
+    <PackageVersion Include="coverlet.collector" Version="6.0.0" />
+    <PackageVersion Include="FluentAssertions" Version="6.12.0" />
+    <PackageVersion Include="MartinCostello.Logging.XUnit" Version="0.3.0" />
+    <PackageVersion Include="Microsoft.Azure.Cosmos" Version="3.36.0" />
+    <PackageVersion Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
+    <PackageVersion Include="MinVer" Version="5.0.0-beta.1" />
+    <PackageVersion Include="xunit.runner.visualstudio" Version="2.5.3" />
+    <PackageVersion Include="xunit" Version="2.6.1" />
+    <PackageVersion Include="Scrutor" Version="4.2.2" />
+  </ItemGroup>
+  <!-- .NET 7.0. bits -->
```

Comment:
I think I spotted an issue the other day when using the package in azure functions, do you think it's worth adding a .net 6 target and downgrading some of the MS packages when we are targeting net6.0?

Reply:
.NET 6 is supported for less than a year remaining, https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle. I personally don't think it's worth doing... especially since we have support for `netstandard2.0` which allows us to use this all over.